modules/github-bots/sdk/github.go

```diff
@@ -585,6 +585,27 @@ func (c GitHubClient) GetFileContent(ctx context.Context, owner, repo, path, ref
 	return content, nil
 }
 
+// SearchContentInFilenameInRepository searches for a text in a filename in a specific repository
+func (c GitHubClient) SearchContentInFilenameInRepository(ctx context.Context, owner, repo, path, content string, opt *github.ListOptions) (*github.CodeSearchResult, error) {
```

Comment:
the only nit was the name, maybe `InRepository` is not needed? 

i.e.

`SearchContentInFilename`

Reply:
Done